1/README.md

```diff
@@ -3,16 +3,17 @@ domain: rfc.tango-controls.org
 shortname: 1/TANGO
 name: Tango Control System
 status: raw
-editor: Vincent Hardion (vincent.hardion@maxiv.lu.se)
+editor: Lorenzo Pivetta (lorenzo.pivetta@elettra.eu)
+contributor: Andy Götz (andy.gotz@esrf.fr)
 ---
 
-This document describes Tango, a control system framework. This RFC define the specification of a control system following the philosophy of Tango.
+This document describes Tango, a control system framework. This RFC define the specification of a control system implementing the philosophy of Tango.
 
 See also: Y/OtherTemplate
 
 ## Preamble
 
-Copyright (c) 2019 MAX IV Laboratory.
+Copyright (c) 2019 European Synchrotron (ESRF).
```

Comment:
Shouldn't this read "Copyright (c) 2019 Tango Controls" ?

Reply:
Right!

Reply:
Damm you discovered my secret plan!